packages/meditrak-server/src/apiV2/surveyResponse.js

```diff
@@ -85,8 +92,8 @@ function buildResponseRecord(user, entitiesByCode, body) {
     user_id: user.id,
     entity_id: entityId || entitiesByCode[entityCode].id,
     data_time: stripTimezoneFromDate(time),
-    start_time: time,
-    end_time: time,
+    start_time: inputStartTime ? inputStartTime : time,
+    end_time: inputEndTime ? inputEndTime : time,
```

Comment:
Ah I was going to make a comment on the default timestamp change for this 😄 .
Do we also want to convert `inputStartTime` and `inputEndTime` to ISO string (similar to `time`, line 87) to keep everything consistent?

Reply:
Good point, updated